analyzers/src/SonarAnalyzer.CSharp/Rules/NewGuidShouldNotBeUsed.cs

```diff
@@ -33,27 +35,56 @@ namespace SonarAnalyzer.Rules.CSharp
     public sealed class NewGuidShouldNotBeUsed : SonarDiagnosticAnalyzer
     {
         internal const string DiagnosticId = "S4581";
-        private const string MessageFormat = "Use 'Guid.NewGuid()' or 'Guid.Empty' or add arguments to this Guid instantiation.";
+        private const string MessageFormat = "Use 'Guid.NewGuid()' or 'Guid.Empty' or use Guid instantiation with arguments.";
```

Comment:
I am not the reviewer so, I just popped up out of nowhere 😃. The message in the RSPEC repo should be also updated for the rule if we changed the message format here.

Reply:
I am planning to mention this today.  😄 
I want to propose changing the RPSEC description a bit and adding more examples.

Reply:
Sounds good! If there is nothing unclear, I think you can add a PR directly and put it in review.

Reply:
Regarding the message, what do you think about the following: `Use 'Guid.NewGuid()', 'Guid.Empty' or the constructor with arguments.`

Reply:
Voila: https://github.com/SonarSource/rspec/pull/590